app/lib/server/functions/setUserActiveStatus.js

```diff
@@ -41,13 +42,17 @@ export function setUserActiveStatus(userId, active, confirmRelinquish = false) {
 	// Users without username can't do anything, so there is no need to check for owned rooms
 	if (user.username != null && !active) {
 		const subscribedRooms = getSubscribedRoomsForUserWithDetails(userId);
+		// give omnichannel rooms a special treatment :)
+		const chatSubscribedRooms = subscribedRooms.filter(({ t }) => t !== 'l');
+		const livechatSubscribedRooms = subscribedRooms.filter(({ t }) => t === 'l');
 
-		if (shouldRemoveOrChangeOwner(subscribedRooms) && !confirmRelinquish) {
-			const rooms = getUserSingleOwnedRooms(subscribedRooms);
+		if (shouldRemoveOrChangeOwner(chatSubscribedRooms) && !confirmRelinquish) {
+			const rooms = getUserSingleOwnedRooms(chatSubscribedRooms);
 			throw new Meteor.Error('user-last-owner', '', rooms);
 		}
 
-		relinquishRoomOwnerships(user._id, subscribedRooms, false);
+		closeOmnichannelConversations(user._id, livechatSubscribedRooms);
```

Comment:
```suggestion
		closeOmnichannelConversations(user, livechatSubscribedRooms);
```

Based on the method definition of `closeOmnichannelConversations`, we should be passing the entire user object here